src/bashi/results.py

```diff
@@ -792,3 +795,19 @@ def _remove_unsupported_cmake_versions_for_clangcuda(
             value_name2=CMAKE,
             value_version2=">3.18",
         )
+        
+        
+def _remove_all_ROCm_images_non_Ubuntu2004_based(
+        parameter_value_pairs: List[ParameterValuePair],
+        removed_parameter_value_pairs: List[ParameterValuePair],
+):
+    remove_parameter_value_pairs(
+        parameter_value_pairs,
+        removed_parameter_value_pairs,
+        value_name1=UBUNTU,
+        parameter1=UBUNTU,
```

Comment:
```suggestion
        parameter1=UBUNTU,
        value_name1=UBUNTU,
```
Please swap the arguments. This is also the default ordering of the function call.

Reply:
This is still valid. Please switch the position of the arguments in both function calls.